client/src/components/util/graphql/Queries.ts

```diff
@@ -19,7 +19,7 @@ export const ITEM_EDIT_GET_ITEM = gql`
 
 export const ITEM_EDIT_GET_ITEMS = gql`
     query {
-        items {
+        item {
```

Comment:
something seems fishy about this query; it kinda seems unused.  You can remove this if IntelliJ says it's unused, otherwise, can you try and figure out what component uses it and make sure that's not broken?

Reply:
Removed this query as it was duplicate to the `ALL_ITEMS` query and replaced any of it's usages as seen in `ItemEditForm.tsx`